pgmuvi/lightcurve.py

```diff
@@ -328,6 +328,41 @@ def __init__(self, xdata, ydata, yerr=None,
         self.__FITTED_MAP = False
         self.__FITTED_MCMC = False
 
+    @classmethod
+    def from_table(cls, VOTable, **kwargs):
+        """Instantiate a Lightcurve object with
+        data read in from a VOTable.
+
+        Parameters
+        ----------
+        VOTable : str
```

Comment:
This could also be a pathlib.Path instance, instead of a string. We should point that out (and use it throughout the docs) since it is more portable.

Reply:
Fixed.